code/modules/urist/admin/verbs.dm

```diff
@@ -1,23 +1,25 @@
 /client/proc/warpallplayers()
-
 	set name = "Warp All Players"
 	set category = "Fun"
 	set desc = "Warp all players to you."
+	set waitfor = FALSE
+
 	if(!check_rights(R_FUN))
 		to_chat(src, "<span class='danger'> You do not have the required admin rights.</span>")
 		return
 
 	for(var/mob/living/M in GLOB.player_list)
-		M.loc = get_turf(usr)
-		message_admins("[key_name(usr)] has warped all players to their location.")
-		log_admin("[key_name(src)] has warped all players to their location.")
+		M.forceMove(get_turf(usr))
+		sleep(-1)
+
+	log_and_message_admins(" has warped all players to their location.")
```

Comment:
Minor nit-pick, but I'd personally prefer this log to be _before_ the `for` loop. If there's any lag it'll give a better indication as to what has caused it. Also iirc you don't need that leading space within the string, that's already added in the proc